tests/osaka/eip7692_eof_v1/eip7620_eof_create/test_legacy_eof_creates.py

```diff
@@ -132,3 +141,55 @@ def test_legacy_initcode_eof_contract_fails(
     )
 
     state_test(env=env, pre=pre, post=post, tx=tx)
+
+
+@pytest.mark.parametrize(
+    "legacy_create_opcode",
+    [
+        pytest.param(Op.CREATE, id="CREATE"),
```

Comment:
I don't think `id` is needed, `Op`s are converted to meaningful names. In this case they are going to be exactly `CREATE` and `CREATE2`.